src/components/changing-confirm/ChangeConfirm.styles.ts

```diff
@@ -1,6 +1,12 @@
 import { TypographyVariantEnum } from '~/types'
 
 export const styles = {
+  root: {
+    width: '684px',
+    maxWidth: '684px',
+    height: '520px',
+    paddingTop: '64px'
```

Comment:
a small reminder. we decided to keep the following pattern in the whole project: pt, pr, mt, mr and etc.